src/plays/2048/Game2048Components/GameOverlay.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import "../Game2048Styles/game-overlay.css"
+import gameOver from "../GameAudio/gameOver.wav"
+import gameWin from "../GameAudio/gameWin.wav"
+
+function GameOverlay(props) {
+
+  const over = new Audio(gameOver)
+  const win = new Audio(gameWin)
+
+  { props.status === "over" ? over.play() : over.pause() }
+  { props.status === "win" ? win.play() : win.pause() }
+  return (
+    <div>
+      <div
+        className={
+          props.status === "over"
```

Comment:
@Shivam-Katare , Can we take this check in a variable and use it in all the places.

Example:
```js
const isStatusOver= props.status === "over"

const isStatusWon= props.status === "won"

```